panic: runtime error: invalid memory address or nil pointer dereference

**Problem with doctl registry repository**

When i run this command i get a title message

```bash
doctl registry repository list-v2 --access-token=dop_*********
``` 

**Environment:**

 - doctl version: doctl version 1.97.0-release
Git commit hash: 0e49bd7a

 - OS: UBUNTU 22.04
 - Installation method: Snap

**Additional Details:**

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x58 pc=0xa1b174]

goroutine 1 [running]:
github.com/digitalocean/doctl/commands/displayers.(*RepositoryV2).KV(0xc000863080)
	github.com/digitalocean/doctl/commands/displayers/registry.go:147 +0xb4
github.com/digitalocean/doctl/commands/displayers.DisplayText({0x1299f18, 0xc000863080}, {0x128f240, 0xc00012e008}, 0x0, {0x0, 0x0, 0xc00099fb50?})
	github.com/digitalocean/doctl/commands/displayers/output.go:91 +0x2eb
github.com/digitalocean/doctl/commands/displayers.(*Displayer).Display(0xc00099fbb8)
	github.com/digitalocean/doctl/commands/displayers/output.go:61 +0x18d
github.com/digitalocean/doctl/commands.(*CmdConfig).Display(0xc0000071e0, {0x1299f18?, 0xc000863080?})
	github.com/digitalocean/doctl/commands/command_config.go:226 +0x135
github.com/digitalocean/doctl/commands.displayRepositoriesV2(...)
	github.com/digitalocean/doctl/commands/registry.go:758
github.com/digitalocean/doctl/commands.RunListRepositoriesV2(0xc0000071e0)
	github.com/digitalocean/doctl/commands/registry.go:623 +0x119
github.com/digitalocean/doctl/commands.cmdBuilderWithInit.func1(0xc000241800?, {0xc000278eb0, 0x0, 0x1})
	github.com/digitalocean/doctl/commands/command.go:85 +0xe4
github.com/spf13/cobra.(*Command).execute(0xc000241800, {0xc000278e90, 0x1, 0x1})
	github.com/spf13/cobra@v1.7.0/command.go:944 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x19995a0)
	github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.7.0/command.go:992
github.com/digitalocean/doctl/commands.Execute()
	github.com/digitalocean/doctl/commands/doit.go:145 +0x27
main.main()
	./main.go:24 +0x31
```

--- a/commands/displayers/registry.go
+++ b/commands/displayers/registry.go
@@ -16,6 +16,7 @@ package displayers
 import (
 	"fmt"
 	"io"
+	"time"
 
 	"github.com/digitalocean/doctl/do"
 )
@@ -143,17 +144,25 @@ func (r *RepositoryV2) KV() []map[string]interface{} {
 	out := make([]map[string]interface{}, 0, len(r.Repositories))
 
 	for _, reg := range r.Repositories {
+		var latestManifest string
 		latestTag := "<none>" // default when latest manifest has no tags
-		if len(reg.LatestManifest.Tags) > 0 {
-			latestTag = reg.LatestManifest.Tags[0]
+		var latestUpdate *time.Time
+
+		if reg.LatestManifest != nil {
+			latestManifest = reg.LatestManifest.Digest
+			if len(reg.LatestManifest.Tags) > 0 {
+				latestTag = reg.LatestManifest.Tags[0]
+			}
+			latestUpdate = &reg.LatestManifest.UpdatedAt
 		}
+
 		m := map[string]interface{}{
 			"Name":           reg.Name,
-			"LatestManifest": reg.LatestManifest.Digest,
+			"LatestManifest": latestManifest,
 			"LatestTag":      latestTag,
 			"TagCount":       reg.TagCount,
 			"ManifestCount":  reg.ManifestCount,
-			"UpdatedAt":      reg.LatestManifest.UpdatedAt,
+			"UpdatedAt":      latestUpdate,
 		}
 
 		out = append(out, m)

--- a/integration/registry_repo_list_v2_test.go
+++ b/integration/registry_repo_list_v2_test.go
@@ -82,6 +82,7 @@ var (
 	repositoryListV2Output = `
 Name      Latest Manifest                                                            Latest Tag    Tag Count    Manifest Count    Updated At
 repo-1    sha256:cb8a924afdf0229ef7515d9e5b3024e23b3eb03ddbba287f4a19c6ac90b8d221    v1            57           82                2021-04-09 23:54:25 +0000 UTC
+repo-2                                                                               <none>        57           82                <nil>
 `
 
 	repositoryListV2Response = `{
@@ -117,10 +118,16 @@ repo-1    sha256:cb8a924afdf0229ef7515d9e5b3024e23b3eb03ddbba287f4a19c6ac90b8d22
           }
         ]
       }
+    },
+    {
+      "registry_name": "example-no-latest-manifest",
+      "name": "repo-2",
+      "tag_count": 57,
+      "manifest_count": 82
     }
   ],
   "meta": {
-    "total": 1
+    "total": 2
   }
 }`
 )

